types/fundamental/real/read.C

```diff
@@ -0,0 +1,59 @@
+#include <ROOT/REntry.hxx>
+#include <ROOT/RNTupleReader.hxx>
+
+using ROOT::Experimental::REntry;
+using ROOT::Experimental::RNTupleReader;
+
+#include <fstream>
+#include <ios>
+#include <ostream>
+#include <string>
+#include <string_view>
+
+template <typename T>
+static void PrintRealValue(const REntry &entry, std::string_view name,
+                           std::ostream &os, bool last = false) {
+  T value = *entry.GetPtr<T>(name);
+  os << "    \"" << name << "\": \"" << value << "\"";
+  if (!last) {
+    os << ",";
+  }
+  os << "\n";
+}
+
+void read(std::string_view input = "types.fundamental.real.root",
+          std::string_view output = "types.fundamental.real.json") {
```

Comment:
Should we also upload the reference files?

Reply:
Eventually yes, once a first version of the RNTuple Validation Suite is complete. That's at least what I wrote in https://github.com/root-project/rntuple-validation/blob/main/README.md#reference-implementation. Until then, I would prefer not to increase the repository size, there will probably be changes until we consider it final.

Reply:
On the other hand, adding the ref file one by one reduce might also make sense.  Verification of the content is (likely to be) easier when the writing of the script is fresh in memory. (I.e. later on technically, we will have to re-read carefully the description before vetting the reference file).

It's okay (i.e. feel free to resolve this conversation) if we don't put them in now but I would have uploaded them.